test/framework/src/main/java/org/elasticsearch/test/rest/yaml/ESClientYamlSuiteTestCase.java

```diff
@@ -111,33 +111,19 @@ public void initAndResetContext() throws Exception {
         if (restTestExecutionContext == null) {
             assert adminExecutionContext == null;
             assert blacklistPathMatchers == null;
-            ClientYamlSuiteRestSpec restSpec = ClientYamlSuiteRestSpec.load(SPEC_PATH);
+            final ClientYamlSuiteRestSpec restSpec = ClientYamlSuiteRestSpec.load(SPEC_PATH);
             validateSpec(restSpec);
-            List<HttpHost> hosts = getClusterHosts();
-            RestClient restClient = client();
-            Version infoVersion = readVersionsFromInfo(restClient, hosts.size());
-            Version esVersion;
-            try {
-                Tuple<Version, Version> versionVersionTuple = readVersionsFromCatNodes(restClient);
-                esVersion = versionVersionTuple.v1();
-                Version masterVersion = versionVersionTuple.v2();
-                logger.info("initializing yaml client, minimum es version: [{}] master version: [{}] hosts: {}",
-                        esVersion, masterVersion, hosts);
-            } catch (ResponseException ex) {
-                if (ex.getResponse().getStatusLine().getStatusCode() == 403) {
-                    logger.warn("Fallback to simple info '/' request, _cat/nodes is not authorized");
-                    esVersion = infoVersion;
-                    logger.info("initializing yaml client, minimum es version: [{}] hosts: {}", esVersion, hosts);
-                } else {
-                    throw ex;
-                }
-            }
-            ClientYamlTestClient clientYamlTestClient = initClientYamlTestClient(restSpec, restClient, hosts, esVersion);
+            final List<HttpHost> hosts = getClusterHosts();
+            Tuple<Version, Version> versionVersionTuple = readVersionsFromCatNodes(adminClient());
```

Comment:
Note here that we use the admin client instead of the plain client. I *think* this means that we do not need the try/catch block anymore.